src/views/Assignments/AssignmentColumns.tsx

```diff
@@ -0,0 +1,238 @@
+import { NewsvalueMap } from '@/defaults/newsvalueMap'
+import { Newsvalue } from '@/components/Table/Items/Newsvalue'
+import { Briefcase, Clock3Icon, Clock9Icon, Crosshair, Navigation, SignalHigh, Users } from '@ttab/elephant-ui/icons'
+import { Newsvalues } from '@/defaults/newsvalues'
+import { FacetedFilter } from '@/components/Commands/FacetedFilter'
+import { AssignmentTypes } from '@/defaults/assignmentTypes'
+import { Type } from '@/components/Table/Items/Type'
+import { getNestedFacetedUniqueValues } from '@/components/Filter/lib/getNestedFacetedUniqueValues'
+import { Assignees } from '@/components/Table/Items/Assignees'
+import { AssignmentTitles } from '@/components/Table/Items/AssignmentTitles'
+import { Actions } from '@/components/Table/Items/Actions'
+import { Tooltip } from '@ttab/elephant-ui'
+import { dateInTimestampOrShortMonthDayTimestamp } from '@/lib/datetime'
+import { type ColumnDef } from '@tanstack/react-table'
+import { type DefaultValueOption } from '@/types/index'
+import { type IDBAuthor } from 'src/datastore/types'
+import {
+  type MetaValueType,
+  type AssignmentMeta,
+  type AssigneeMeta
+} from './types'
+import { slotLabels, timesSlots } from '@/defaults/assignmentTimeslots'
+
+export function assignmentColumns({ authors = [], locale, timeZone }: {
+  authors?: IDBAuthor[]
+  locale: string
+  timeZone: string
+}): Array<ColumnDef<AssignmentMeta & { planningTitle: string, newsvalue: string, _id: string }>> {
+  return [
+    {
+      id: 'titles',
+      meta: {
+        name: 'Titlar',
+        columnIcon: Briefcase,
+        className: 'flex-1'
+      },
+      accessorFn: ({ planningTitle, title: assignmentTitle }) => {
+        return {
+          planningTitle,
+          assignmentTitle
+        }
+      },
+      cell: ({ row }) => {
+        const data: { planningTitle: string, assignmentTitle: string } = row.getValue('titles') || {}
+        const { assignmentTitle, planningTitle } = data
+        return <AssignmentTitles planningTitle={planningTitle} assignmentTitle={assignmentTitle} />
+      }
+    },
+    {
+      id: 'newsvalue',
+      meta: {
+        Filter: ({ column, setSearch }) => (
+          <FacetedFilter column={column} setSearch={setSearch} />
+        ),
+        options: Newsvalues,
+        name: 'Nyhetsvärde',
+        columnIcon: SignalHigh,
+        className: 'box-content w-4 sm:w-8 pr-1 sm:pr-4'
+      },
+      accessorFn: ({ newsvalue }) => {
+        return newsvalue
+      },
+      cell: ({ row }) => {
+        const value: string = row.getValue('newsvalue') || ''
+        const newsvalue = NewsvalueMap[value]
+
+        if (newsvalue) {
+          return <Newsvalue newsvalue={newsvalue} />
+        }
+      },
+      filterFn: (row, id, value) => {
+        return value.includes(row.getValue(id))
+      }
+    },
+    {
+      id: 'assignees',
+      meta: {
+        options: authors.map((_) => ({ value: _.name, label: _.name })),
```

Comment:
Might be better to use `a` instead of `_` as Gustav just made me aware that it is common to use `_` for parameters that aren't used (typescript doesn't even complain on unused parameters when using it).

Reply:
Pushed new commit to fix this.